src/screens/ReportCar.js

```diff
@@ -12,8 +25,20 @@ const ReportCar = ({ navigation, props, route }) => {
       <View style={styles.container}>
         {imageURI && <Image style={styles.image} source={{ uri: imageURI }} />}
         <Text>Report Him!</Text>
-        <Button onPress={()=>navigation.navigate("Camera")} title={'Take a photo'}/>
-        <Button onPress={()=>navigation.navigate("Photo")} title={'Select photo'}/>
+        <Button
+          onPress={() => navigation.navigate("Camera")}
+          title={"Take a photo"}
+        />
+        <Button
+          onPress={() => navigation.navigate("Photo")}
+          title={"Select photo"}
+        />
+        {/* Poniższy 'przycisk' jest do susunięcia, 
+          doodany tylko w celu testowania metody dodającej token do bazy */}
+        <Button onPress={() => insert("x")} title={"Add"} />
+        {/* Poniższy 'przycisk' jest do susunięcia, 
+          doodany tylko w celu testowania metody spr czy dany token jest w bazie */}
+        <Button onPress={() => pri()} title={"Find"} />
```

Comment:
Please remove that code, it doesn't make sense to put code related to finding token in reportCar form